libs/application/template-api-modules/src/lib/modules/templates/financial-statement-individual-election/financial-statement-individual-election.service.ts

```diff
@@ -57,17 +56,12 @@ export class FinancialStatementIndividualElectionService extends BaseTemplateApi
       return Promise.reject({})
     }
 
-    const { bucket, key } = AmazonS3Uri(fileName)
-
     try {
-      const file = await this.s3
-        .getObject({
-          Bucket: bucket,
-          Key: key,
-        })
-        .promise()
-      const fileContent = file.Body as Buffer
-      return fileContent.toString('base64') || ''
+      const fileContent = await this.s3Service.getFileContent(
+        fileName,
+        'base64',
+      )
+      return fileContent || ''
```

Comment:
This is a good point, but this would change the functionality of the submit application function where this is called. It would no longer "silently fail" when the filename is not in the application attachments dictionary, which might not be what we want anyway, but thats not the focus of this PR